beacons-data/beacons_bulk_load_legacy.rb

```diff
@@ -0,0 +1,278 @@
+		secondaryOwners = []
+
+		Faker::Number.between(from: 1, to: 3).times do |secondaryOwnerIndex|
+			owner_name = Faker::Name.name
+			main_use = "N"
+			secondaryOwners.insert(secondaryOwnerIndex,buildOwner(main_use,owner_name,created_date,last_modified_date))
+		end
+
+		data = {
+			"uses": uses,
+			"owner": primaryOwner,
+			"beacon": buildBeacon(hex_id,created_date,last_modified_date),
+			"secondaryOwners": secondaryOwners,
+			"emergencyContact": {
+				"details": " #{owner_name}, TEL: #{Faker::PhoneNumber.phone_number}\n\nMATCOM F TEL: #{Faker::PhoneNumber.phone_number}   FAX: #{Faker::PhoneNumber.phone_number}\n\nVessel Email: #{owner_email}"
+			}
+		}.to_json
+
+		#puts data
+
+		#debug_output = "INSERT INTO legacy_beacon (id, hex_id, owner_email, owner_name, created_date, beacon_status, data, last_modified_date, use_activities) VALUES (#{uuid}, #{hex_id}, #{owner_email}, #{owner_name}, #{created_date}, #{beacon_status}, #{data}, #{last_modified_date}, #{use_activities});"
+
+		puts hex_id
+
+		# run SQL
+		conn.exec_prepared('statement', [ uuid, hex_id, owner_email, owner_name, created_date, beacon_status, data, last_modified_date, use_activities])
+	end
+end
+
+def buildBeaconUse(environment, activity, main_use,created_date,last_modified_date)
+
+	call_sign = Faker::Artist.name
+	vessel_name = Faker::Artist.name
+	more_details = Faker::Movies::StarWars.quote
+	purpose = ["PLEASURE", "COMMERCIAL"].sample
+	position = "#{Faker::Address.latitude} #{Faker::Address.longitude}"
+	tripInfo = Faker::Movies::BackToTheFuture.quote
+
+	# Maritime only fields
+	if environment == "MARITIME"
+		homePort = Faker::Address.city
+		vessel_type = purpose
+		beacon_location = ["Cabin", "Hull", "On Life Jacket", "Backpack"].sample
+	end
+
+	#Aircraft only fields
+	if environment == "AIRCRAFT"
+		aircraft_type = purpose
+		aircraft_description = "#{Faker::Color.color_name} #{aircraft_type}"
+		aircraft_registration_mark = Faker::Vehicle.vin
+		principal_airport = Faker::Address.city
+		beacon_location = ["Cabin", "Under Seat", "Hold"].sample
+	end
+
+	#Rig/Platform only fields
+	if environment == "RIG/PLATFORM"
+		rigName = Faker::Movies::StarWars.vehicle
+		beacon_location = ["On Platform", "ON Rig"].sample
+	end
+
+	#Land only fields
+	if environment == "LAND"
+		landUse = activity
+		beacon_location = ["In Backpack", "On Person", "In Vehicle"].sample
+	end
+
+	#MOD only fields
+	if environment == "MOD"
+		beacon_location = ["Classified Location","Redacted"].sample
+		mod_type = Faker::Vehicle.model
+		mod_status = ["ACTIVE","INACTIVE"].sample
+		mod_variant = Faker::Vehicle.manufacture
+	end
+
+	return {
+		"note": Faker::Movies::StarWars.quote,
+		"notes": $default_value,
+		"isMain": main_use,
+		"landUse": landUse,
+		"rigName": rigName,
+		"useType": activity,
+		"callSign": call_sign,
+		"homePort": homePort,
+		"position": position,
+		"tripInfo": tripInfo,
+		"areaOfUse": $default_value,
```

Comment:
I think for these $defaultValue fields it would be nice to populate them with something so they're not blank